Add Video Calling/iOSVideo/ContentView.swift

```diff
@@ -0,0 +1,368 @@
+            let scalingMode = ScalingMode.fit
+            localVideoStream = LocalVideoStream(camera: camera)
+            previewRenderer = try! VideoStreamRenderer(localVideoStream: localVideoStream!)
+            previewView = try! previewRenderer!.createView(with: RenderingOptions(scalingMode:scalingMode))
+            call!.startVideo(stream:localVideoStream) { (error) in
+                if (error != nil) {
+                    print("cannot start video")
+                }
+                else {
+                    self.sendingVideo = true
+                }
+            }
+        }
+    }
+
+    func startCall() {
+        guard let deviceManager = deviceManager else {
+            return
+        }
+        let camera = deviceManager.cameras.first
+        self.localVideoStream = LocalVideoStream(camera: camera)
+        let videoOptions = VideoOptions(localVideoStream: localVideoStream)
+                    
+        let scalingMode = ScalingMode.fit
+        self.previewRenderer = try! VideoStreamRenderer(localVideoStream: self.localVideoStream!)
+        self.previewView = try! self.previewRenderer?.createView(with: RenderingOptions(scalingMode: scalingMode))
+        self.sendingVideo = true
+        print("Sending video", self.sendingVideo)
+                    
+        let startCallOptions = StartCallOptions()
+        startCallOptions?.videoOptions = videoOptions
+  
+        let callees:[CommunicationIdentifier] = [CommunicationUserIdentifier(self.callee)]
+        self.call = self.callAgent?.startCall(participants: callees, options: startCallOptions)
+        
+        self.callObserver = CallObserver(self)
+        self.call!.delegate = self.callObserver
+                    
+        self.remoteParticipantObserver = RemoteParticipantObserver(self)
+    }
+
+    func endCall() {
+        self.call!.hangUp(options: HangUpOptions()) { (error) in
+            if (error != nil) {
+                print("ERROR: It was not possible to hangup the call.")
+            }
+        }
+        self.previewRenderer?.dispose()
+        self.remoteRenderer?.dispose()
+        sendingVideo = false
+    }
+}
+
+public class RemoteVideoStreamData : NSObject, RendererDelegate {
+    public func rendererFailedToStart() {
+        owner.errorMessage = "Renderer failed to start"
+    }
+    
+    private var owner:ContentView
+    let stream:RemoteVideoStream
+    var renderer:VideoStreamRenderer? {
+        didSet {
+            if renderer != nil {
+                renderer!.delegate = self
+            }
+        }
+    }
+    var views:[RendererView] = []
+    init(view:ContentView, stream:RemoteVideoStream) {
+        owner = view
+        self.stream = stream
+    }
+    public func onFirstFrameRendered() {
+        let size:StreamSize = renderer!.size
+        owner.remoteVideoSize = String(size.width) + " X " + String(size.height)
+    }
+}
+
+public class CallObserver: NSObject, CallDelegate, IncomingCallDelegate {
```

Comment:
Lets move all the Observers to one file and rename IncomingCallHandler.swift to something more generic